### ISSUE TITLE
Error handling

### ISSUE THREAD

**Issue by no-longer-on-githu-b** [purescript-node/purescript-node-http#7]
It seems there is no way to attach error handlers to HTTP servers. Am I missing something?

E.g. the equivalent of the following JS:

```
var s = http.createServer(h);
s.on('error', function(err) { console.log('oops!', err); });
s.listen(9000); // in case of failure, will call the above callback
```

**Comment by hdgarrood** [comment]
I don't think you're missing anything, I think it is just that nobody has needed that yet so there's no binding for it. I guess we want an `onError :: Server -> (Error -> Eff _ Unit) -> Eff _ Unit` in Node.HTTP?

**Comment by no-longer-on-githu-b** [comment]
Yeah that should work. Shall I do a PR tonight?

**Comment by hdgarrood** [comment]
Go for it. :)

**Comment by JordanMartinez** [comment]
Fixed by #49.
